lapack/impl/KokkosLapack_svd_impl.hpp

```diff
@@ -0,0 +1,34 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#ifndef KOKKOSLAPACK_IMPL_SVD_HPP_
+#define KOKKOSLAPACK_IMPL_SVD_HPP_
+
+/// \file KokkosLapack_svd_impl.hpp
+/// \brief Implementation(s) of dense linear solve.
```

Comment:
Should be "Implementation(s) of singular value decomposition of a dense matrix."

Reply:
Good point x)